search-index-import-util/src/test/java/no/unit/nva/publication/DataPipelineFileReaderPublicationTest.java

```diff
@@ -0,0 +1,120 @@
+package no.unit.nva.publication;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.ListObjectsV2Request;
+import com.amazonaws.services.s3.model.ListObjectsV2Result;
+import com.amazonaws.services.s3.model.S3Object;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import no.unit.nva.search.exception.ImportException;
+import no.unit.nva.search.exception.SearchException;
+import no.unit.nva.utils.ImportDataRequest;
+import nva.commons.utils.Environment;
+import nva.commons.utils.IoUtils;
+import nva.commons.utils.log.LogUtils;
+import nva.commons.utils.log.TestAppender;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class DataPipelineFileReaderPublicationTest {
```

Comment:
DataPipelineFileReaderPublicationTest returns something but there is no test for the happy scenario.

Write the tests for what you want DataPipelineFileReaderPublication without thinking what it already does and how it does it.

Reply:
removed DataPipelineFileReaderPublication class completely, was a leftover to be used for datamigration